packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -358,6 +361,17 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
     super.forceUpdate();
   }
 
+  /**
+   * Get the current height the list and it's pages.
+   */
+  public getTotalListHeight(): number {
+    let height = 0;
```

Comment:
Rather than iterating, wouldn't `_surfaceRect.height` fit the purpose?

Reply:
Yes, that seems to work.  I just did some tests to confirm that they return the same value.  I will put the change in the next iteration.

Reply:
I have put this change in.